doc/2/guides/getting-started/4-authentication/index.md

```diff
@@ -4,4 +4,105 @@ type: page
 title: Authentication
 description: Authenticate users
 order: 400
----
\ No newline at end of file
+---
+
+# Authentication
+
+Kuzzle authentication system is multi-strategy based. This means that the same user can authenticate in several different ways.
+
+For example, the same user can authenticate with the `local` strategy with an username and a password pair but also with the `oauth` strategy using an external provider such as Facebook or Google.
+
+::: info
+Kuzzle uses [Passport.js](http://www.passportjs.org/packages/) under-the-hood and therefore there is 300+ strategies that are made available. (LDAP, Active Directory, x509, etc.)  
+See [how to integrate a new strategy](/core/2/some-link-on-integrating-new-strategy)
+:::
+
+We saw that in the [Access Control Rights](/core/2/guides/getting-started/3-access-control-rights) guide. When creating a user, we had to provide credentials for the `local` strategy but we could have provided more strategies:
+
+```bash
+$ kourou security:createUser '{
+  content: {
+    profileIds: ["dummy"]
+  },
+  credentials: {
+    local: {
+      username: "yagmur",
+      password: "password"
+    },
+    ldap: {
+      bindDN: "cn=root",
+      searchBase: "ou=passport-ldapauth",
+      searchFilter: "(uid=yagmur)"
+    }
+  }
+}'
+```
+
+### Get an authentication token
+
+Kuzzle uses authentication token.  
+
+First we need to get one with the [auth:login](/core/2/api/controllers/auth/login) action. This action takes the `strategy` used to authenticate and any information needed by the strategy.
+
+In our example we will use the `local` strategy so we have to provide an `username` and a `password`:
+
+```bash
+$ kourou auth:login -a strategy=local -a username=yagmur -a password=password
+[ℹ] Unknown command "auth:login", fallback to API method
+ 
+ 🚀 Kourou - Executes an API query.
+ 
+ [ℹ] Connecting to http://localhost:7512 ...
+ {
+  "_id": "62843356-d826-42fb-adf1-e930e90b6006",
+  "expiresAt": 1602600225701,
+  "jwt": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI2Mjg0MzM1Ni1kODI2LTQyZmItYWRmMS1lOTMwZTkwYjYwMDYiLCJpYXQiOjE2MDI1OTY2MjUsImV4cCI6MTYwMjYwMDIyNX0.0HZF_AhyTzPCRxdaMbT6hlwLflYG4emmLlTD6YV_Nmo",
+  "ttl": 3600000
+ }
+ [✔] Successfully executed "auth:login"
+```
+
+Kuzzle sent us back the token in the `jwt` property
+
+::: warning
+When executing the [auth:login](/core/2/api/controllers/auth/login) action, the anonymous user rights will apply.
+
+So if the anonymous user does not have the rights to execute this action, then no one will be able to login.
```

Comment:
Kuzzle prevents the `auth:login` right from being removed from the `anonymous` role. If you try to remove it, you get a `security.rights.login_required` error.

Also, the first sentence is slightly inexact, since it is possible to perform an `auth:login` as a logged in user (if their associated roles allow it)

```suggestion
Usually, login attempts are made by anonymous users, to acquire a token granting necessary rights to perform more actions.

Since removing rights to the `auth:login` action from anonymous users would mean that it would be no longer possible to log in, Kuzzle prevents that action from ever be removed from the `anonymous` role.
```

Reply:
Do we ? Preventing this action to be removed from anonymous role?